c7n/resources/rds.py

```diff
@@ -31,12 +31,14 @@
            op: ne
 
 """
+from __future__ import annotations
```

Comment:
I don't _think_ any of the changes here require this import - did it sneak in through some autofix during local editing perhaps?